src/utils/thread_helper.cpp

```diff
@@ -944,6 +944,43 @@ void ThreadHelper::DetachGracefullyCallback(void)
 
 #if OTBR_ENABLE_TELEMETRY_DATA_API
 #if OTBR_ENABLE_BORDER_ROUTING
+void ThreadHelper::RetrieveInfraLinkInfo(threadnetwork::TelemetryData::InfraLinkInfo *aInfraLinkInfo)
```

Comment:
How about using a reference instead of a pointer here? (Since we don't anticipate or handle the case where the passed-in input pointer is `nullptr`.)

Reply:
sg! Thanks for the suggestions!